functions-and-operators/string-functions.md

```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
 
 ### [`UCASE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ucase)
 
-Synonym for `UPPER()`.
+The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT UCASE('bigdata') AS result_upper, UCASE('null') AS result_null;
```

Comment:
To keep it consistent with the example for `UPPER()` https://github.com/pingcap/docs/pull/16534/files#r1502244394
```suggestion
SELECT UCASE('bigdata') AS result_upper, UCASE(null) AS result_null;
```